### PR TITLE
QA-4138

### PR DESCRIPTION
**Pull request by Berkof**
Add usage of the local Maven repository: https://ggtc.gridgain.com/buildConfiguration/Dev_Qa_YCSBRunCI/9435825?hideProblemsFromDependencies=false&hideTestsFromDependencies=false

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@ LICENSE file.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>site.ycsb</groupId>
+  <groupId>org.gridgain.ycsb</groupId>
   <artifactId>root</artifactId>
   <version>0.18.0-SNAPSHOT</version>
```
Comment (nsivkov) [inline pom.xml:24]: Do we need to set an internal version like `2023.8`?

File: asynchbase/pom.xml [hunk 2]
```diff
@@ -47,7 +47,7 @@ LICENSE file.
         </exclusion>
       </exclusions>
     </dependency>
-    
+
```
Comment (ahitrin) [inline asynchbase/pom.xml:50]: I don't think we should include changes like formatting or whitespace removal into our 3rd party forks. They always make further upstream integration much more difficult.

File: pom.xml (hunk 1 above)
Comment (ahitrin) [reply pom.xml:24]: +1 for using custom version number

**Review by ahitrin — APPROVED** [review]
Approved with minors

**Pull request merged**